packages/core/application/core-application-browser-internal/src/utils/append_app_path.ts

```diff
@@ -8,7 +8,7 @@
 
 import { removeSlashes } from './remove_slashes';
 
-export const appendAppPath = (appBasePath: string, path: string = '') => {
+export const appendAppPath = (appBasePath = '', path: string = '') => {
```

Comment:
I checked, and all usages today seem to provide valid strings, so this change should be harmless at the moment (unless a `.js` file starts using it in the future)